packages/runtime/container-runtime/src/gc/garbageCollection.ts

```diff
@@ -904,6 +913,24 @@ export class GarbageCollector implements IGarbageCollector {
 			lastSummaryTime: this.getLastSummaryTimestampMs(),
 			viaHandle: requestHeaders?.[RuntimeHeaders.viaHandle],
 		});
+
+		// We may throw when loading an Inactive object, depending on these preconditions
+		const shouldThrowOnInactiveLoad =
+			!this.isSummarizerClient &&
+			this.configs.throwOnInactiveLoad === true &&
+			requestHeaders?.[AllowInactiveRequestHeaderKey] !== true;
+		const state = this.unreferencedNodesState.get(nodePath)?.state;
+
+		if (shouldThrowOnInactiveLoad && reason === "Loaded" && state === "Inactive") {
+			const request: IRequest = { url: nodePath };
```

Comment:
The `nodePath` here will be the data store's id even when a DDS is requested because that's what the container runtime passes. That will make it harder to distinguish between data store and DDS requests.
The `nodeUpdated` function should accept the full `request` from `getDataStoreFromRequest` and log it here (and in telemetry tracker).

Reply:
From my observation when adding a DDS test and stepping through the code, nodePath is actually the full path.

Reply:
Right, looks like the request is passed not the id. That's good then :)